microservices/configcheck/src/test/resources/input/microservice/QueryLogicFactory.xml

```diff
@@ -114,6 +114,10 @@
         <!-- the zookeeper configuration (could be something like file:///opt/datawave/zookeeper-warehouse/conf/zoo.cfg, or the zookeeper list (somehost1:2181,somehost2:2181,…)-->
         <property name="zookeeperConfig" value="${datawave.query.logic.logics.BaseEventQuery.zookeeperConfig}" />
 
+        <!-- This is injected via BaseEventQueryConfiguration.java and configured via configuration properties -->
+        <!-- the list of directories in which the query service puts its caches (selection always made on query service side) -->
+        <property name="localIvaratorCacheDirConfigs" ref="baseEventQueryLocalIvaratorCacheDirConfigs" />
```

Comment:
plug for future rename:  since "local" is relative, maybe simply call them what they are.  webserverIvaratorCacheDirConfigs and tserverIvaratorCacheDirConfigs